### PR TITLE
perf: use `hash` to replace `createHash`

### PR DESCRIPTION
**Pull request by btea**
### Description
`crypto.hash` seems to be faster than `crypto.createHash`.

https://nodejs.org/docs/latest-v20.x/api/crypto.html#cryptohashalgorithm-data-outputencoding)

I did a simple test and `crypto.hash` seems to be 1.6 times faster.

image

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. If the feature is substantial or introduces breaking changes without a discussion, PR might be closed.
- [ ] Ideally, include a test that fails without this PR but passes with it.
- [ ] Please, don't make changes to `pnpm-lock.yaml` unless you introduce a new test example.

### Tests
- [ ] Run the tests with `pnpm test:ci`.

### Documentation
- [ ] If you introduce new functionality, document it. You can run documentation with `pnpm run docs` command.

### Changesets
- [ ] Changes in changelog are generated from PR name. Please, make sure that it explains your changes in an understandable manner. Please, prefix changeset messages with `feat:`, `fix:`, `perf:`, `docs:`, or `chore:`.

### CODE REVIEW

**Review by sheremet-va — DISMISSED** [review]

File: packages/vitest/src/utils/index.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import crypto from 'node:crypto'
```
Comment (sheremet-va) [inline packages/vitest/src/utils/index.ts:1]: Ah, actually. This file can be imported in any environment. The utility should be in node-specific utils file

Comment (btea) [reply packages/vitest/src/utils/index.ts:1]: Should I create a separate file or does such a file already exist and I need to move this function into it?

Comment (sheremet-va) [reply packages/vitest/src/utils/index.ts:1]: Creating a separate file should be fine

Comment (btea) [reply packages/vitest/src/utils/index.ts:1]: Ok.

**Review by sheremet-va — APPROVED** [review]

**Pull request merged**